Cargo.toml

```diff
@@ -21,6 +21,8 @@ regex = "1"
 smallvec = { version="1.6", features=["union"] }
 thiserror = "1.0"
 tree-sitter = "0.20"
+serde = "1.0.117"
+serde_json = "1.0.72"
```

Comment:
Is there anything in these particular patch versions that you're depending on, or could you list the requirement as `1.0` in both cases?

Reply:
Looks like they both loosen just fine 👍🏻